appservice/thirdpartynotices.txt

```diff
@@ -276,3 +276,187 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 END OF request-promise NOTICES AND INFORMATION
 ==================================
+
+websocket NOTICES BEGIN HERE
```

Comment:
Almost - you have to add this to the "table of contents" at the top of this file as well